### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by koppor**
Enjoy!

### CODE REVIEW

File: src/main/resources/l10n/Menu_de.properties [hunk 1]
```diff
@@ -1,28 +1,29 @@
+#X-Generator: crowdin.com
 Abbreviate\ journal\ names\ (ISO)=Zeitschriftentitel abkürzen (&ISO)
 Abbreviate\ journal\ names\ (MEDLINE)=Zeitschriftentitel abkürzen (&MEDLINE)
 About\ JabRef=Über &JabRef
 Append\ library=Bibliothek &anhängen
 Autogenerate\ BibTeX\ keys=&BibTeX-Keys automatisch generieren
 Close\ library=Bibliothek s&chließen
-Copy=&Kopieren
```
Comment (Siedlerchr) [inline src/main/resources/l10n/Menu_de.properties]: Seems like we loose our mnemonic keys!

Comment (koppor) [reply src/main/resources/l10n/Menu_de.properties]: I try to work on it. Therefore, you see these pull requests. Please ignore them until I say, it is OK!

Comment (koppor) [reply src/main/resources/l10n/Menu_de.properties]: The issue is that I loaded some wrong translations and they are kept somewhere at Crowdin.

Comment (koppor) [reply src/main/resources/l10n/Menu_de.properties]: Note that we currently also loose the German translation. So please be patient!

**Pull request closed without merging**